lib/ldap_fluff/generic_member_service.rb

```diff
@@ -70,7 +70,7 @@ def get_logins(userlist)
 
   def get_login_from_entry(entry)
     [@attr_login, 'uid', 'cn'].each do |attribute|
-      return entry.send(attribute) if entry.respond_to? attribute
+      return entry.send(attribute.map(&:downcase)) if entry.respond_to? attribute
```

Comment:
In my case, my username is uppercase in AD, and it works with this patch.

To remove the downcase here, we should probably remove other downcase calls in ldap_fluff. I'm afraid it could break things.